packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -71,13 +81,13 @@ export class OzonContentFiguurNode implements OzonContentNode {
             }
           : undefined;
 
-      return (
+      const ozonFiguur = (
         <div class={`dso-ozon-figuur ${bijschrift ? `bijschrift-${bijschrift.locatie}` : "onder"}`}>
           {titel && <span class="figuur-titel">{titel}</span>}
           {bijschrift?.locatie === "boven" && (
             <Bijschrift bijschrift={bijschrift} bron={bron} mapNodeToJsx={mapNodeToJsx} />
           )}
-          <dso-image-overlay>
+          <dso-image-overlay editAction={editAction} editActionLabel={editActionLabel}>
```

Comment:
Heb je overwogen om het `editActionLabel` in Image Overlay te resolven?

Reply:
Nee.